Sources/BraintreeCore/Analytics/BTAnalyticsService.swift

```diff
@@ -53,7 +53,7 @@ class BTAnalyticsService: Equatable {
         linkType: String? = nil,
         payPalContextID: String? = nil
     ) async {
-        let timestampInMilliseconds = UInt64(Date().timeIntervalSince1970 * 1000)
+        let timestampInMilliseconds = Int(round(Date().timeIntervalSince1970 * 1000))
```

Comment:
nit: It might be good to keep `UInt64` explicit here, but at the same time there aren't any [32-bit iOS devices](https://stackoverflow.com/a/27440158) out there anymore so it'll implicitly be a 64-bit number. 32-bit timestamps are the new [Y2K bug](https://www.linkedin.com/pulse/2038-next-y2k-bug-legacy-time-overflow-issues-john-o-shea-948pf) for some machines.

Reply:
Interesting! Yeh I think Apple will cover us here, according to their [docs for `Int`](https://developer.apple.com/documentation/swift/int#overview)

>  On 32-bit platforms, Int is the same size as Int32, and on 64-bit platforms, Int is the same size as Int64.